server/controllers/reviews.js

```diff
@@ -0,0 +1,36 @@
+const Reviews = require('../models/Reviews');
+const asyncHandler = require('express-async-handler');
+
+exports.getAllReviews = asyncHandler(async (req, res) => {
+  try {
+    const review = await Reviews.find({
+      artistId: req.user.id,
```

Comment:
Won't this give us reviews only for the logged in user?
We are supposed to get all the reviews for a specific artist, so we should be using a path parameter here to get the artist's id.